### PR TITLE
added meta-data in serverside props

### PR DESCRIPTION
**Pull request by tejassonar**
Added meta data in Server-side props for better SEO

@norbertschuler

### CODE REVIEW

File: pages/index.tsx [hunk 1]
```diff
@@ -247,6 +251,7 @@ export async function getServerSideProps(context: any) {
       donationStep = 0;
     }
   }
+  const resolvedUrl = context.resolvedUrl;
```
Comment (norbertschuler) [inline pages/index.tsx:254]: Is it correct that http://localhost:3000/?to=proj_6x3GTD5cMRv0OeQAiIlJZ0Au results into e.g. `<meta property="twitter:image" content="https://s.wordpress.com/mshots/v1/http%3A%2F%2Flocalhost%3A3000%2F%3Fto%3Dproj_6x3GTD5cMRv0OeQAiIlJZ0Au?w=1200&amp;h=770.jpg">`

**Review by norbertschuler — APPROVED** [review]
Seems ok for me.

File: pages/index.tsx [hunk 2]
```diff
@@ -113,44 +115,41 @@ function index({
...
+  //     }. Your journey to a trillion trees starts here.`;
+  //   } else if (projectDetails.purpose === "bouquet") {
+  //     description = `Make a contribution to ${projectDetails.name}. ${
+  //       projectDetails.description ? projectDetails.description : ""
+  //     } Your journey to a trillion trees starts here.`;
+  //   }
+  // }
+  // if (giftDetails && giftDetails.recipientName) {
+  //   title = `Join ${giftDetails.recipientName} - Donate with Plant-for-the-Planet`;
+  // }
```
Comment (norbertschuler) [inline pages/index.tsx:142]: I vote for removing all commented code.

File: pages/index.tsx (hunk 1 above)
Comment (sarveshpro) [reply pages/index.tsx:228]: https://s.wordpress.com/mshots/v1/https%3A%2F%2Fdonate-with-planet-m434g86p1-planetapp.vercel.app%2F%3Fto%3Dproj_6x3GTD5cMRv0OeQAiIlJZ0Au?w=1200&amp;h=770.jpg

I checked the image in the preview link of this PR it works. Or maybe I didn't understand your question.

**Review by sagararyal — APPROVED** [review]

**Pull request merged**